modules/rosa-deleting-cluster.adoc

```diff
@@ -9,24 +9,109 @@ endif::[]
 
 :_content-type: PROCEDURE
 [id="rosa-deleting-cluster_{context}"]
-= Deleting a cluster
+ifndef::sts[]
+= Deleting a ROSA cluster
+endif::sts[]
+ifdef::sts[]
+= Deleting a ROSA cluster and the cluster-specific IAM resources
+endif::sts[]
 
-You can delete an {product-title} cluster using the `rosa` CLI.
+ifndef::sts[]
+You can delete a {product-title} cluster using the rosa CLI.
```

Comment:
For this reference to the rosa CLI, should it be one of the following (for consistency):
ROSA CLI (`rosa`)
`rosa` CLI